pkg/repository/cachedep.go

```diff
@@ -57,7 +57,7 @@ func (r *record) Exec(newTx bool) error {
 	if r.cacheEntry.ID <= 0 {
```

Comment:
has to be adjusted: `Exec(conn db.Connection)` is correct, the current implementation is wrong

Reply:
Fixed